src/components/map/Map.tsx

```diff
@@ -1,64 +1,83 @@
+import VectorTileLayer from './VectorTileLayer.js';
+import CountriesTileLayer from "./CountriesTileLayer"
+import { layerStyle } from './MapStyle';
+
 
 export default function Map() {
-  const mapRef = createRef<LeafletMap>();
-  const geoJsonRef = createRef<GeoJSON>();
   const [state] = useContext(AppContext);
-  const [mapRecords, setMapRecords] = useState(Countries as any);
-  const [forceUpdate, setForceUpdate] = useState(Math.random());
-  const buckets = getBuckets(mapRecords.features);
+  const [mapRecords, setMapRecords] = useState(state.countries as any);
+  const buckets = getBuckets(mapRecords);
 
   useEffect(() => {
-    const importGeo = Countries as any;
-    const features = importGeo.features as GeoJSON.Feature[]
-    // We will iterate through all the features in the geoJson
-    // if they are in the country dict we will attach their aggregated data to the feature for displaying
-    importGeo.features = mapAltDataToFeatures(features, state.explore.estimateGradePrevalences);
-    setMapRecords(importGeo)
+    if(state.explore.estimateGradePrevalences.length > 0){
+      const countriesData = state.countries.map( (country : any) => {
+        const countryEstimate = state.explore.estimateGradePrevalences.find(element => element.alpha3Code === country.alpha3Code);
 
-    // we need to update the key on the GEOJSON to let react know it's time to rerender
-    setForceUpdate(Math.random())
-  }, [state.explore.estimateGradePrevalences])
+        if (countryEstimate && countryEstimate.testsAdministered) {
+          const { testsAdministered, geographicalName, numberOfStudies, localEstimate, nationalEstimate, regionalEstimate, sublocalEstimate } = countryEstimate;
+          return { ...country, properties: { testsAdministered, geographicalName, numberOfStudies, localEstimate, nationalEstimate, regionalEstimate, sublocalEstimate }}
+        }
+        return { ...country, properties: { testsAdministered: null, geographicalName: null, numberOfStudies: null, localEstimate: null, nationalEstimate: null, regionalEstimate: null, sublocalEstimate: null } }
+      })
+      setMapRecords(countriesData);
+    }
+  }, [state.explore.estimateGradePrevalences, state.countries])
 
   const bounds = latLngBounds([-90, -200], [90, 180]);
   const maxBounds = latLngBounds([-90, -200], [90, 200]);
 
   const mapboxAccessToken = process.env.REACT_APP_MAPBOX_API_KEY;
+
   return (
-    <LeafletMap
-      ref={mapRef}
-      center={[0, 0]}
-      zoom={1}
+    <MapContainer
+      center={bounds.getCenter()}
+      zoom={3}
       className="map w-100"
       bounceAtZoomLimits={true}
       bounds={bounds}
-      minZoom={2}
+      minZoom={3}
       maxBounds={maxBounds}
-      enableHighAccuracy={true}
       maxBoundsViscosity={1}
     >
+
+      <CountriesTileLayer
+        url="https://tiles.arcgis.com/tiles/5T5nSi527N4F7luB/arcgis/rest/services/Countries/VectorTileServer"
+        records={mapRecords}
+        zIndex={50}
```

Comment:
Should we put all our zIndex in some sort of constants file? Don't want someone to accidentally update a zIndex and break our UI down the road.

Reply:
We should absolutely do this

Reply:
The zindex in the map container are separate from all other elements on the page but I'll make the change